examples/hitl/rearrange_v2/rearrange_v2.py

```diff
@@ -365,11 +367,32 @@ def pop_ui_events(self) -> List[Dict[str, Any]]:
         return events
 
     def _get_camera_lookat_pos(self) -> mn.Vector3:
+        # HACK: Estimate camera height.
+        # TODO: GuiController should have knowledge on the agent type it controls, and should provide an API to get the camera height.
+        gui_agent_controller = self.gui_agent_controller
+        sim = self.app_service.sim
+        agent_data = self.agent_data
+        camera_height_offset = 1.0
+        if isinstance(gui_agent_controller, GuiRobotController):
```

Comment:
@zephirefaith : Beware that until this code is handled at the right place, you'll have to check for `LLMController`.